src/sentry/static/sentry/app/views/organizationIntegrations/pluginDetailedView.tsx

```diff
@@ -0,0 +1,332 @@
+> {
+  componentDidMount() {
+    const {location} = this.props;
+    const value =
+      location.query.tab === 'configurations' ? 'configurations' : 'information';
+
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({tab: value});
+  }
+
+  getEndpoints(): ([string, string, any] | [string, string])[] {
+    const {orgId, pluginSlug} = this.props.params;
+    return [
+      ['plugins', `/organizations/${orgId}/plugins/configs/?plugins=${pluginSlug}`],
+    ];
+  }
+  get plugin() {
+    return this.state.plugins[0];
+  }
+
+  handleResetConfiguration = (projectId: string) => {
+    //make a copy of our project list
+    const projectList = this.plugin.projectList.slice();
+    //find the index of the project
+    const index = projectList.findIndex(item => item.projectId === projectId);
+    //should match but quit if it doesn't
+    if (index < 0) {
+      return;
+    }
+    //remove from array
+    projectList.splice(index, 1);
+    //update state
+    this.setState({
+      plugins: [{...this.state.plugins[0], projectList}],
+    });
+  };
+
+  handleEnablePlugin = (projectId: string) => {
+    //make a copy of our project list
+    const projectList = this.plugin.projectList.slice();
+    //find the index of the project
+    const index = projectList.findIndex(item => item.projectId === projectId);
+    //should match but quit if it doesn't
+    if (index < 0) {
+      return;
+    }
+
+    //update item in array
+    projectList[index] = {
+      ...projectList[index],
+      enabled: true,
+    };
+
+    //update state
+    this.setState({
+      plugins: [{...this.state.plugins[0], projectList}],
+    });
+  };
+
+  handleAddToProject = () => {
+    const plugin = this.plugin;
+    const {organization, router} = this.props;
+    openModal(
+      ({closeModal, Header, Body}) => (
+        <ContextPickerModal
+          Header={Header}
+          Body={Body}
+          nextPath={`/settings/${organization.slug}/projects/:projectId/plugins/${plugin.id}/`}
+          needProject
+          needOrg={false}
+          onFinish={path => {
+            closeModal();
+            router.push(path);
+          }}
+        />
+      ),
+      {}
+    );
+  };
+
+  onTabChange = (value: Tab) => {
+    this.setState({tab: value});
+  };
+
+  featureTags() {
+    return this.plugin.features.map(feature => (
+      <StyledTag key={feature}>{feature.replace(/-/g, ' ')}</StyledTag>
+    ));
+  }
+
+  mapPluginToProvider() {
+    const plugin = this.plugin;
+    return {
+      key: plugin.slug,
+    };
+  }
+
+  renderBody() {
+    const plugin = this.plugin;
+    const {tab} = this.state;
+    const {organization} = this.props;
+
+    // Prepare the features list
+    const features = plugin.featureDescriptions.map(f => ({
+      featureGate: f.featureGate,
+      description: (
+        <FeatureListItem
+          dangerouslySetInnerHTML={{__html: singleLineRenderer(f.description)}}
+        />
+      ),
+    }));
```

Comment:
Do we have to use `dangerouslySetInnerHTML` because the descriptions have actual HTML tags in them? If so, isn't it risky to allow users to write HTML that would show up on Sentry?

Reply:
It's markdown, the markdown parser prevents HTML

Reply:
@leeandher Good point, I actually don't think in this case we need to use that here. But just so you know, `singleLineRenderer` will actually sanitize the inputs so it wouldn't be risky.

Reply:
Ahh okay, thanks guys